CONTRIBUTING.md

```diff
@@ -53,18 +53,15 @@ If you need a public release (for example, to run it on your project CI), you ca
 Note that only core maintainers (Remoters) can publish public releases. If needed, ask us in the PR and we'll do it for you. Check #3 for the video walkthrough.
 
 1.  Locally run the script `npm run release:dev:patch` or `npm run release:dev:minor` depending on your changes.
-    a. You'll be shown what's the new version and prompt you if it's correct. Eg
-
-        ```
-        Creating a new dev...
-        :: Current version: 1.0.0
-        :::::: New version: 1.0.1-dev.20230516175718
-        Ready to commit and publish it? (y/n)
-
-        ```
-
-    b. Then it will ask for the `@remoteoss` OTP for the NPM access to publish it.
-    c. Done! 🎉
+    1. You'll be shown what's the new version and prompt you if it's correct. Eg
```

Comment:
note: Go check `main`. These nested lists are totally broken. To work it must be always numeric, instead of alphabetic.

Reply:
this looks a lot better, letsgo